operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -1753,6 +1792,80 @@ def isSSLListenPortEnabled(ssl, domain):
       enabled = True
   return enabled
 
+def getSSLPortIfEnabled(server, domain, is_server_template=True):
+  """
+  return the SSL listen port if enabled -
+    If SSL is enabled:
+      If is_server_template is False then just return the SSL listen port from server mbean.
+      If is_server_template is True then return the actual SSL listen port that it listens on.  If the server
+
+    If SSL is not enabled but domain has SecureMode enabled return 7002.
```

Comment:
SSL is not enabled -> SSL is not configured.

If SSL is configured but explicitly disabled, return None.

Reply:
fixed